diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+wdi grace try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+It is an open source program that allows me to track changes to files.
```

Comment:
This is what Git is.  GitHub is a user interface for accessing git's version control capabilities for our project repositories.